cmd/src/teams_members_add.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+Examples:
+
+  Add a team member:
+
+    	$ src teams members add -team-name='engineering' [-email='alice@sourcegraph.com'] [-username='alice'] [-id='VXNlcjox'] [-external-account-service-id='https://github.com/' -external-account-service-type='github' [-external-account-account-id='123123123'] [-external-account-login='alice']]
```

Comment:
TODO: I would love if this supported adding multiple users at once, like the API already supports, but I didn't find a nice way in command flags to specify multiple users, as all of these flags might relate to the same user, but also might now. If someone has ideas, I'm all ears :)